Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -1059,7 +1054,9 @@ static ErrorOr<void> read_huffman_table(JPEGStream& stream, JPEGLoadingContext&
         TRY(table.generate_codes());
 
         auto& huffman_table = table.type == 0 ? context.dc_tables : context.ac_tables;
-        huffman_table.set(table.destination_id, table);
+        auto& table_register = table.type == 0 ? context.registered_dc_tables : context.registered_ac_tables;
+        huffman_table[table.destination_id] = table;
```

Comment:
While I'm leaving a comment, this should certainly be `move(table)`?

Reply:
It was a move() I think, but this is a memory anyways, no?

Reply:
> but this is a memory anyways

did you mean memcpy?
I don't think it can be because it contains several heap-allocated Vectors. 

(I also vaguely remember there being a size limit to memcpy-able copies, and this thing is 592 bytes, but I also could have made that one up in my head).